_pylibczi/Image.cpp

```diff
@@ -0,0 +1,126 @@
+//
+// Created by Jamie Sherman on 2019-08-28.
+//
+
+#include <algorithm>
+#include <numeric>
+#include <typeinfo>
+#include <utility>
+#include <iostream>
+
+#include "Image.h"
+#include "Iterator.h"
+#include "exceptions.h"
+#include "helper_algorithms.h"
+
+namespace pylibczi {
+
+  std::unique_ptr<std::map<libCZI::PixelType, std::string> > ImageBC::s_pixelToTypeName(
```

Comment:
as a global var, could you just do away with the unique_ptr and just make it a static std::map?